app/components/brand_header_component.html.erb

```diff
@@ -0,0 +1,9 @@
+<div style='background-color: <%= @space.brand_header_color %> ' class="flex flex-col md:flex-row flex-wrap justify-between sm:px-5 py-5 items-center">
```

Comment:
It would be nice if we used CSS for this by doing something like:

```erb
<!-- This lives in layout.html.erb -->
<style>
.SpaceHeader {
  background-color: <%= @space.header_background_color %>
}
</style>

<!-- this stays here -->
<div class="SpaceHeader">
...
</div>
```

Reply:
I like _class-based_ css way better than Tailwind style utility classes, but I think mixing those patterns will get confusing.